tests/app/views/test_marketplace.py

```diff
@@ -399,8 +400,39 @@ def test_can_apply_to_live_brief_with_new_supplier_flow(self):
     def test_cannot_apply_to_closed_brief(self):
         brief = self.brief.copy()
         brief['briefs']['status'] = "closed"
-        brief['briefs']['publishedAt'] = "2000-01-25T12:00:00.000000Z"
-        brief['briefs']['applicationsClosedAt'] = "2000-02-25T12:00:00.000000Z"
+        brief['briefs']['publishedAt'] = "2016-12-01T12:00:00.000000Z"
```

Comment:
Maybe same here on the comment about published at after the feature flag. It does seem like a bit silly to do so but it took me quite a while to get my head around the tests going on and I've spent plenty of time on the context so hopefully it will be worth it at a later point.